### ISSUE TITLE
Make leaderboard page as table

### ISSUE THREAD

**Issue by luxaritas** [eternagame/eternagame.org#145]
Columns aren’t aligned and isn’t properly mobile responsive. Should be reimplemented as a table (also better for accessibility)

**Comment by agarg5** [comment]
Update: Columns not aligned, but still need to reimplement as table for better accessibility
